src/context/TxContext/index.tsx

```diff
@@ -64,16 +63,20 @@ function TxContextProvider({children}: PropTypes): React.ReactElement {
 
     modalRef.current?.open();
 
+    const {meta} = transaction.registry.findMetaCall(transaction.callIndex);
+    const args = meta?.args.map(({name}) => name).join(', ') || '';
+    const title = `Sending transaction ${section}.${method}(${args})`;
+    const description = formatCallMeta(meta);
```

Comment:
What does the output of this look like? is it user friendly? Is it displayed somewhere?

Reply:
yes this is the same as before for transactions, but instead of giving it explicitly it is derived from the api. it looks like this currently in the app.
![Screenshot 2021-08-10 at 10 26 40](https://user-images.githubusercontent.com/1168983/128834237-4979d9c3-005a-4738-babb-e6a7166cadc3.png)